webserver/views/index.py

```diff
@@ -62,3 +62,7 @@ def goals():
 @index_bp.route("/faq")
 def faq():
     return render_template("index/faq.html")
+
+@index_bp.route("/down")
```

Comment:
Maybe it'd be better to do this directly with nginx. You can use something like `try_files` (http://nginx.org/en/docs/http/ngx_http_core_module.html#try_files) to see if a file exists (is_down.html), and serve that, otherwise forward to the uwsgi server. We use this workflow in some other projects to put the app into maintenance mode while we upgrade/restart the app server.